projects/views.py

```diff
@@ -261,10 +257,7 @@ def view_project(request, project_id):
                         )
                 except Exception:
```

Comment:
Not sure if it's in the scope of this patch, but might be nice to have an additional catch for `User.DoesNotExist`. I imagine this could get frustrating with an easy-to-miss typo.

Reply:
We already catch this above: https://github.com/ChameleonCloud/portal/pull/269/files#diff-cec7e4e74af6d0ee95b13c06b15ad41ec82fac8041923a65a5a58fc270669f3bL227